x/bank/types/types.proto

```diff
@@ -54,3 +48,25 @@ message MsgMultiSend {
   repeated Input  inputs  = 1 [(gogoproto.nullable) = false];
   repeated Output outputs = 2 [(gogoproto.nullable) = false];
 }
+
+// ModuleAccount defines an account for modules that holds coins on a pool
+message ModuleAccount {
```

Comment:
should this be moved to auth?

Reply:
Yeah I was thinking that it probably makes sense to.

Reply:
did you decide to keep it in bank?